qt-bugs@ issue : none
Trolltech task ID : none
applied: yes
author: David Faure <faure@kde.org>

Needed to distinguish qt-copy from released versions of qt, at least while
we backport API additions to qt-copy.

Index: src/corelib/global/qglobal.h
===================================================================
--- src/corelib/global/qglobal.h        (revision 536662)
+++ src/corelib/global/qglobal.h        (working copy)
@@ -38,6 +38,11 @@
 #include <QtCore/qconfig.h>
 #endif

+/* QT_KDE_QT_COPY is set in qt-copy so that we know when we can use features backported to qt-copy
+   Keep in mind that it should also work with the right release of Qt, so use something like:
+      #if defined(QT_KDE_QT_COPY) || QT_VERSION >= 0x040200 */
+#define QT_KDE_QT_COPY
+
 /*
    The operating system, must be one of: (Q_OS_x)

